auto_gptq/modeling/_base.py

```diff
@@ -1220,6 +1194,8 @@ def skip(*args, **kwargs):
                         safe_save(new_state_dict, model_save_name)
 
             if use_marlin:
+                if is_sharded:
```

Comment:
why ？

Reply:
@xunfeng1980 it is just not tested/implemented, but should be.